internal/view/table.go

```diff
@@ -95,6 +95,19 @@ func (t *Table) SendKey(evt *tcell.EventKey) {
 
 func (t *Table) keyboard(evt *tcell.EventKey) *tcell.EventKey {
 	key := evt.Key()
+
+	// Handle Ctrl+Left/Right for column selection
+	if evt.Modifiers()&tcell.ModCtrl != 0 {
```

Comment:
On my box Ctrl-R|L is a mission control shortcut to switch desktops (I think that a default on osx??). 
This breaks the header column nav ;(
Might be best to use `Shift <|>` to cycle the sort instead?